django_slack/backends.py

```diff
@@ -71,5 +71,22 @@ def send(self, *args, **kwargs):
         self._send.delay(*args, **kwargs)
 
 
+class TestBackend(Backend):
```

Comment:
Adding a quick docstring along the lines of "this is for testing, use retrieve message" would make this perfect.